Resources/public/templates/tabs/details.hbt

```diff
@@ -0,0 +1,29 @@
+<div>
+    <p>Creator: {{contentCreator.name}} - {{content.publishedDate}}</p>
```

Comment:
when the users are not yet loaded you should display a loader to the user as it's done in the other AsynchronousView usage in the application